tests/unit/monitor/alerts/group_alerts_by_table/mock_classes.py

```diff
@@ -0,0 +1,30 @@
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Dict, List, Optional, Union
+
+
+@dataclass
+class MockAlert:
+    status: Optional[str]  # should be in "warn", "error" , "fail[ure]"
+    slack_group_alerts_by: Optional[str]
+    model_unique_id: Optional[str]
+    slack_channel: Optional[str]
+    detected_at: Optional[Union[str, datetime]]
+    model_meta: Optional[
+        str
+    ]  # this string should be a json of a dict that has or has not the key "channel"
+    owners: Optional[List[str]]
+    subscribers: Optional[List[str]]
+    tags: Optional[str]
+
+
+@dataclass
+class MockConfig:
+    slack_group_alerts_by: Optional[str]
+    slack_channel_name: Optional[str]
+
+
+@dataclass
+class MockDataMonitoringAlerts:
```

Comment:
I think that it could be nice if you'll create a Mock for DataMonitoringAlerts that inheritance from DataMonitoringAlerts (like we do with the fetchers and the apis), and move it to the global mocks folder.
WDYT?

Reply:
I thought this is an over kill for the unit testing, but would be reqiured anyway for the E2E testing